src/components/Text.tsx

```diff
@@ -73,7 +74,12 @@ export const TextWithLineBreaks = ({ text }: { text: string }) => (
       <p key={x}>
         {x.split(/[\n\r]/g).map((y, i, arr) => (
           <React.Fragment key={y}>
-            {y}
+            <span
+              // eslint-disable-next-line react/no-danger
+              dangerouslySetInnerHTML={{
+                __html: DOMPurify.sanitize(y),
+              }}
+            />
```

Comment:
this turns on markup for all `Text`. No concerns?

Reply:
None for now.